src/others/helpers/aid-request-helpers.ts

```diff
@@ -0,0 +1,89 @@
+import { groupBy, map, filter, eq, every } from "lodash";
+import { Location, Supply, AidRequest, ID } from "../contexts/api";
+
+import { ListItem } from "../components/CollapsibleListItem";
+
+export const translateToLocation =
+  (cities: Location[]) =>
+  (city_id: number): Location => {
+    return cities[city_id - 1];
```

Comment:
why do we need to hardcode this logic in one more place? 
why the one in DictionaryContext cannot be used instead of this?

Reply:
We'll need it to process data consumed by map, temporarily. I'd added a comment explaining this.

Reply:
Two more things I don't get:
1) why there is "city_id - 1"? why -1?
2) what is the point of this translateToLocation at all if one could use locationDict?

Reply:
As a matter of fact it was not needed anymore - deleted. 
Thanks!